senza/templates/postgresapp.py

```diff
@@ -208,6 +221,81 @@
             Resource:
               - {{kms_arn}}
           {{/kms_arn}}
+  SpiloMasterSG:
+    Type: "AWS::EC2::SecurityGroup"
+    Properties:
+      GroupDescription: "Security Group for the master ELB"
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: {{postgres_port}}
+          ToPort: {{postgres_port}}
+          CidrIp: {{elb_access_cidr}}
+  {{#add_replica_loadbalancer}}
+  SpiloReplicaSG:
+    Type: "AWS::EC2::SecurityGroup"
+    Properties:
+      GroupDescription: "Security Group for the replica ELB"
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: {{postgres_port}}
+          ToPort: {{postgres_port}}
+          CidrIp: 0.0.0.0/0
+  {{/add_replica_loadbalancer}}
+  SpiloMemberSG:
+    Type: "AWS::EC2::SecurityGroup"
+    Properties:
+      GroupDescription: "Security Group for Spilo members"
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: {{postgres_port}}
+          ToPort: {{postgres_port}}
+          SourceSecurityGroupId:
+            Fn::GetAtt:
+              - SpiloMasterSG
+              - GroupId
+        - IpProtocol: tcp
+          FromPort: {{healthcheck_port}}
+          ToPort: {{healthcheck_port}}
+          SourceSecurityGroupId:
+            Fn::GetAtt:
+              - SpiloMasterSG
+              - GroupId
+
+        {{#add_replica_loadbalancer}}
+        - IpProtocol: tcp
+          FromPort: {{postgres_port}}
+          ToPort: {{postgres_port}}
+          SourceSecurityGroupId:
+            Fn::GetAtt:
+              - SpiloReplicaSG
+              - GroupId
+        - IpProtocol: tcp
+          FromPort: {{healthcheck_port}}
+          ToPort: {{healthcheck_port}}
+          SourceSecurityGroupId:
+            Fn::GetAtt:
+              - SpiloReplicaSG
+              - GroupId
+        {{/add_replica_loadbalancer}}
+
+        - IpProtocol: tcp
+          FromPort: 0
+          ToPort: 65535
+          SourceSecurityGroupName: "Odd (SSH Bastion Host)"
```

Comment:
It is hardcoded, will convert it into a function during senza init, together with the Security Group for the zmon-agent.